doc/sites.rst

```diff
@@ -96,7 +96,9 @@ site using the :doc:`generate_sites_module stanza
     (sites mygui))
 
 The generated module `mysites` depends on the library `dune-site` provided by
-Dune.
+Dune. If `implicit_transitive_deps` is set to false in your `dune-project`
```

Comment:
I think it would be helpful to have a reference to the `implicit_transitive_deps` stanza in the reference manual.